jiant/tasks/qa.py

```diff
@@ -608,3 +607,102 @@ def preprocess_qasrl_datum(cls, datum):
                 for verb_idx, verb_entry in datum["verbEntries"].items()
             ],
         }
+
+
+@register_task("commonsenseqa", rel_path="CommonsenseQA/")
+@register_task("commonsenseqa-easy", rel_path="CommonsenseQA/", easy=True)
```

Comment:
Why is this called "easy"?

Reply:
The commensenQA comes with a sanity check setting, that's the easy task.